crates/notan_winit/Cargo.toml

```diff
@@ -18,12 +18,14 @@ notan_app = { path = "../notan_app", version = "0.7.1" }
 notan_glow = { path = "../notan_glow", version = "0.7.1" }
 notan_audio = { path = "../notan_audio", version = "0.7.1", optional = true }
 notan_oddio = { path = "../notan_oddio", version = "0.7.1", optional = true }
+notan_input = { path = "../notan_input", version = "0.7.1", optional = true }
```

Comment:
this cuz we need modifier keys state to decide cut/copy/paste

Reply:
It makes sense to me.